### PR TITLE
Got rid of the `IterationMode` template arguments.

### PR DESCRIPTION
**Pull request by grixa**
Iterate<MLS, IM> -> Iterate\<MLS> + IterateUnsafe\<MLS>
PersisterIterateImpl<MLS, IM> -> PersisterIterate\<MLS> + PersisterIterateUnsafe\<MLS>

### CODE REVIEW

File: blocks/ss/persister.h [hunk 1]
```diff
@@ -112,17 +110,26 @@ class EntryPersister : public GenericEntryPersister<ENTRY>, public IMPL {
...
-            IterationMode IM = IterationMode::Safe>
-  IterableRange<IM> Iterate(std::chrono::microseconds from,
+  template <current::locks::MutexLockStatus MLS = current::locks::MutexLockStatus::NeedToLock>
+  IterableRangeUnsafe IterateUnsafe(uint64_t begin = static_cast<uint64_t>(0), uint64_t end = static_cast<size_t>(-1)) const {
+    return IMPL::template PersisterIterateUnsafe<MLS>(begin, end);
+  }
+
+  template <current::locks::MutexLockStatus MLS = current::locks::MutexLockStatus::NeedToLock>
+  IterableRange Iterate(std::chrono::microseconds from,
                             std::chrono::microseconds till = std::chrono::microseconds(-1)) const {
```
Comment (mzhurovich) [inline blocks/ss/persister.h]: `make indent`?

Comment (grixa) [inline blocks/ss/persister.h]: Thanks! Fixed in db3804a

**Comment by dkorolev** [comment]
:+1:

**Comment by mzhurovich** [comment]
@grixa Thanks!

**Pull request merged**